test/studies/lulesh/intelVectBug/lulesh.compopts

```diff
@@ -0,0 +1,6 @@
+-M../../../release/examples/benchmarks/lulesh --fast
```

Comment:
Sorry, by "that would just make the future pass" I mean "decoupling --vectorize from --fast would make the future pass." -- i.e., it wouldn't go unnoticed.